apis/meta/v1alpha1/secrets.go

```diff
@@ -45,7 +45,7 @@ const (
 	SecretTypePostgresql corev1.SecretType = "goharbor.io/postgresql"
 
 	// PostgresqlPasswordKey is the password to connect to postgresql.
-	PostgresqlPasswordKey = "postgresql-password" //nolint:gosec
+	PostgresqlPasswordKey = "postgres-password"
```

Comment:
This changed in the upstream `postgresql` bitnami chart. I fear this is a breaking change for upgrades.